meetings/__init__.py

```diff
@@ -169,7 +176,7 @@ async def endmeeting(self, evt: MessageEvent) -> None:
       await self.send_respond(evt, f'Meeting ended at {time_from_timestamp(evt.timestamp)} UTC', meeting=meeting)
 
       # Do backend-specific endmeeting things
-      await self.backend.endmeeting(self, evt, meeting_id)
+      await self.backend.endmeeting(self, evt, meeting)
```

Comment:
This passes the whole row from the DB, right? My Python is ... not great :P

Reply:
yeah, i needed more information than just the meeting_id in the fedora backend -- not sure if just passing the whole row is the right thing to do here, or manually sending through each col that i need.

Reply:
I think it's fine, you'd only end up doing the same SQL lookup in the backend anyway. I was just checking my understanding :)